### PR TITLE
test bad revisions

### PR DESCRIPTION
**Pull request by jywarren**
re #7062

### CODE REVIEW

File: test/unit/revision_test.rb [hunk 1]
```diff
@@ -15,6 +15,21 @@ class RevisionsTest < ActiveSupport::TestCase
...
   end
+  
+  test 'create a bad revision' do
+    node = Node.new(uid: users(:bob).id,
+                    type: 'page')
+    node.title = 'My new node for revision testing'
+    assert node.save!
+    # in testing, uid and id should be matched, although this is not yet true in production db
+    node_revision = Revision.new(title: 'My new node',
+                                 body: 'I hope this test works -- by which I mean fails! 😅',
```
Comment (jywarren) [inline test/unit/revision_test.rb]: If this causes a test failure, let's try to fix it with https://github.com/publiclab/plots2/pull/7172

```ruby
        # prevent vid non-unique bug in https://github.com/publiclab/plots2/issues/7062
        raise ActiveRecord::Rollback if !node.valid? || node.vid == 0
```

**Comment by jywarren** [comment]
Nope! This did not cause a test failure here, probably for the same mysterious reason that it did not in #3007 ... difference between Travis and our live site containers...

**Comment by SidharthBansal** [comment]
@VladimirMikulic I think you will be willing to take this one too

**Comment by VladimirMikulic** [comment]
@SidharthBansal as always, just tag me :)

**Comment by SidharthBansal** [comment]
Lol

**Pull request closed without merging**